### PR TITLE
ci: dispatch main releasability after merged PRs

### PR DESCRIPTION
**Pull request by sgajbi**
## Summary
- Add a bounded merged-PR dispatcher workflow for `main-releasability.yml`.
- Keep `main-releasability.yml` `workflow_dispatch`-only so the merged-PR dispatcher is the single automatic post-merge path and does not duplicate/cancel push-triggered runs.
- Add workflow contract tests for trigger, permissions, main-branch guard, timeout, dispatch target, and no duplicate push trigger.
- Update README, repo context, operations docs, and wiki source with current release-evidence posture.

Issue: #552

## Validation
- `python -m pytest tests/unit/test_main_releasability_dispatch_workflow.py tests/unit/test_workflow_action_runtime.py -q` => 19 passed
- `python scripts/check_workflow_action_runtime.py` => passed
- `git diff --check` => passed
- `make check` after review fix => 1690 passed
- `make ci` final PR head after review fix => passed; 254 integration tests, 1944 coverage tests, 94.58% total coverage against 84% gate, dependency security audit passed
- `powershell -ExecutionPolicy Bypass -File C:\Users\Sandeep\projects\lotus-platform\automation\Sync-RepoWikis.ps1 -CheckOnly -Repository lotus-gateway -AllowUnpublishedSourceChanges` => expected DiffCount 1 because this branch changes repo-authored wiki source
- `python C:\Users\Sandeep\projects\lotus-platform\automation\validate_branch_commit_budget.py --repo-root . --base-ref origin/main --head-ref HEAD` => within budget, 2 commits

## Governance evidence
- Lane placement: dispatcher is separate from PR Merge Gate and triggers the existing Main Releasability Gate after merged PRs to `main`.
- Deduplication: `main-releasability.yml` no longer has a direct `push` trigger; manual `workflow_dispatch` remains for operator reruns.
- Permissions: dispatcher uses `actions: write`, `contents: read`; no repo contents write permission.
- Stranded truth: `git branch -r --no-merged origin/main` returned no unmerged remote branches for `lotus-gateway` before implementation.
- Wiki: repo-authored wiki source changed; publish after merge with `Sync-RepoWikis.ps1 -Publish -Repository lotus-gateway`, then strict check-only parity.
- Non-degradation: no runtime/API contract behavior changed; existing OpenAPI, migration, integration, coverage, security, and workflow governance gates passed.
- Skill/context decision: no platform skill change in this repo PR; reviewer feedback is captured in sgajbi/lotus-platform#688 to tighten the platform template/validator so future rollouts do not keep both automatic paths.

## Post-merge requirements
- Verify exact-main Main Releasability Gate for the merge SHA.
- Update issue #552 to `status/merged-main` after exact-main proof, wiki publication/parity, and branch cleanup evidence.
- Remove `lotus-gateway` from the platform auto-merge releasability exception register in a follow-up platform PR after this PR lands and exact-main proof passes.

### CODE REVIEW

File: .github/workflows/merged-pr-main-releasability.yml [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+    if: >
+      github.event.pull_request.merged == true &&
+      github.event.pull_request.base.ref == 'main'
+    runs-on: ubuntu-latest
+    timeout-minutes: 10
+    steps:
+      - name: Dispatch main releasability gate
+        env:
+          GH_TOKEN: ${{ github.token }}
+        run: gh workflow run main-releasability.yml --repo "$GITHUB_REPOSITORY" --ref main
```
[the remark this answers is not in the document]
Comment (sgajbi) [reply .github/workflows/merged-pr-main-releasability.yml]: Fixed in `2c7e5429`: `main-releasability.yml` is now `workflow_dispatch`-only and the merged-PR dispatcher is the single automatic post-merge path. Added/updated tests assert no direct `push` trigger remains, and final-head `make check` plus `make ci` passed locally.
